hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -676,13 +676,10 @@ object LoadVCF {
       fatal("arguments refer to no files")
 
     inputs.foreach { input =>
-      if (!input.endsWith(".vcf")
-        && !input.endsWith(".vcf.bgz")) {
-        if (input.endsWith(".vcf.gz"))
-          checkGzippedFile(hConf, input, forceGZ, gzAsBGZ)
-        else
-          fatal(s"unknown input file type `$input', expect .vcf[.bgz]")
-      }
+      if (!(input.endsWith(".vcf") || !input.endsWith(".vcf.bgz") || !input.endsWith(".vcf.gz")))
```

Comment:
I think you don't want the `!` for the second two conditions, right?

Reply:
Right, did not mean to leave that in. Thanks!